easybuild/easyconfigs/l/lddt/lddt-1.2.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'Tarball'
+
+name = 'lddt'
+version = '1.2'
+
+homepage = 'https://openstructure.org'
+description = """The local Distance Difference Test (lDDT) is a superposition-free score which evaluates local distance
+differences in a model compared to a reference structure."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://openstructure.org/static/']
+sources = ['%(name)s-linux.zip']
+checksums = ['3f4f72dc790f64431d29f7ff04353d3bbade8c8c12a487a1a99240a82fc4fcec']
+
+modextrapaths = {'PATH': ''}
+
+sanity_check_paths = {
+    'files': ['lddt', 'stereo_chemical_props.txt'],
+    'dirs': [],
+}
+
```

Comment:
Especially for `Tarball` we should add a `sanity_check_command` to test the provided binary. I think we can use `lddt -e` here.

Reply:
Done, sadly this wont work for the other 2 as they provide no version check